Makefile

```diff
@@ -62,12 +62,13 @@ endif
 VELERO_PLUGIN ?= ${CLUSTER_TYPE}
 
 ifeq ($(CLUSTER_TYPE), ibmcloud)
-	VELERO_PLUGIN ?= aws
+	VELERO_PLUGIN = aws
```

Comment:
So we're not allowing overrides here anymore?

Reply:
changed because was not working for me, VELERO_PLUGIN was being set to ibmcloud

Reply:
ok. Sounds like tech debt todo :)

Reply:
@kaovilai Looking at the code above, we're explicitly settting `VELERO_PLUGIN` to `${CLUSTER_TYPE}` -- so on line 65, it will *always* already be set to "ibmcloud" if `CLUSTER_TYPE` is "ibmcloud". So I think "=" is what we want, not "?=", since any user-set VELERO_PLUGIN will already be overwritten with CLUSTER_TYPE, and also "ibmcloud" is not a valid plugin.

Reply:
in makefile yes we are explicitly setting it to ibmcloud but the [following](https://github.com/kaovilai/oadp-operator/blob/b09ce0461bc113e3dbb409d52e38b46555e808dc/Makefile#L64-L66) should've worked.
```
ifeq ($(CLUSTER_TYPE), ibmcloud)
	VELERO_PLUGIN ?= aws
endif
```

Reply:
Ok I see now, this is nested under ibmcloud cluster_type. I still feel if velero_plugin is unset and cluster_type is aws that 
```
ifeq ($(CLUSTER_TYPE), ibmcloud)
	VELERO_PLUGIN ?= aws
endif
```
should've worked..

resolving.